jules-julesforrest/hw_3/1-object-exercise/monkey.js

```diff
@@ -0,0 +1,56 @@
+/* Work with a partner to create objects for three different monkeys (called monkey1, monkey2, monkey3), each with the following properties:
+
+* name
+* species
+* foodsEaten
+
+And the following methods:
+* eatSomething(food)
+* introduce: produces a string introducing itself, including its name, species, and what it's eaten.
+
+Create 3 monkeys total. Make sure all 3 monkeys have all properties set and methods defined.
+
+Choose which partner will initially take the pair programming role of driver and who will be the observer.
+Switch roles after creating the first object, then again after creating the second.
+Also be sure each partner takes a turn in each role during the exercise steps below.
+
+Exercise your monkeys by retrieving their properties and using their methods.
+Practice using both syntaxes for retrieving properties (dot notation and brackets).
```

Comment:
Get comfortable using both dot notation and brackets for accessing properties (key/value pairs) in objects. Monkey functionality works well otherwise. 👍